ctx.go

```diff
@@ -271,7 +273,7 @@ func (c ContextNoBody) QueryParamArr(name string) []string {
 	if !ok {
 		slog.Warn("query parameter not expected in OpenAPI spec", "param", name)
 	}
-	return c.Req.URL.Query()[name]
+	return c.urlValues[name]
```

Comment:
I know the code was there.

But why not calling Get here ?


```suggestion
	return c.urlValues.Get(name)
```


As the code does a few line below?

Reply:
Returning the whole array. This pattern was discussed in a previous PR where I asked the same question xD

Reply:
See #169

Reply:
Thanks for your reply.

It's clearer